recipes/kmtricks/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+KMER_SIZE_LIST="32 64 96 128 160 192 224 256"
+
+mkdir -p $PREFIX/bin
+
+cmake -S . -B build -DCMAKE_BUILD_TYPE=Release \
+                    -DNATIVE=OFF \
+                    -DCONDA_BUILD=ON \
+                    -DWITH_MODULES=ON \
+                    -DWITH_HOWDE=ON \
+                    -DKMER_LIST="${KMER_SIZE_LIST}" \
+                    -DWITH_SOCKS=ON
+cmake --build ./build
+
+cp -r ./bin/kmtricks $PREFIX/bin
+cp -r ./bin/kmtricks-socks $PREFIX/bin
+
```

Comment:
Can you please add a comment what this is doing?

Reply:
Sorry I meant the new complication and the third binary

Reply:
kmtricks supports plugins through runtime dynamic linking. As this implies an overhead, plugin support is provided in a separate binary, called kmtricks**p**. I will add a comment in the script. Thanks for the review.